orchagent/dash/dashvnetorch.cpp

```diff
@@ -333,12 +333,13 @@ bool DashVnetOrch::addVnetMap(const string& key, VnetMapBulkContext& ctxt)
         {
             addOutboundCaToPa(key, ctxt);
             addPaValidation(key, ctxt);
+            return true;
```

Comment:
I think we want to remove the entries from consumer.m_toSync only after flushing the bulk entries. This is handled later when addVnetMapPost returns true. 

if (addVnetMapPost(key, ctxt))
                {
                    it_prev = consumer.m_toSync.erase(it_prev);
                }
                else
                {
                    it_prev++;
                }

Reply:
The PR is not fully resolving the issue. The changes contradict the logic shown above. 
@prabhataravind I'm closing this PR and will open an issue instead.